app/upgrades.go

```diff
@@ -56,32 +57,20 @@ func (app UmeeApp) RegisterUpgradeHandlers(bool) {
 	app.registerUpgrade("v5.0", upgradeInfo, ugov.ModuleName, wasm.ModuleName)
 	app.registerUpgrade5_1(upgradeInfo)
 	app.registerUpgrade6(upgradeInfo)
-	// app.registerNewTokenEmissionUpgrade(upgradeInfo)
 }
 
-// TODO: this upgrade registration is just for testing purpose, once we finalize the release for new token emission
-// then we need to change planName and storeUpgrades
-// func (app *UmeeApp) registerNewTokenEmissionUpgrade(upgradeInfo upgradetypes.Plan) {
-// 	// TODO:finalize the name
-// 	planName := "token_emission"
-// 	app.UpgradeKeeper.SetUpgradeHandler(planName,
-// 		func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-// 			return app.mm.RunMigrations(ctx, app.configurator, fromVM)
-// 		},
-// 	)
-
-// 	app.storeUpgrade(planName, upgradeInfo, storetypes.StoreUpgrades{
-// 		Added:   []string{metoken.ModuleName},
-// 		Deleted: []string{"gravity"},
-// 	})
-// }
-
 func (app *UmeeApp) registerUpgrade6(upgradeInfo upgradetypes.Plan) {
 	planName := "v6.0"
 	gravityModuleName := "gravity" // hardcoded to avoid dependency on GB module
 
 	app.UpgradeKeeper.SetUpgradeHandler(planName,
 		func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			if err := upgradev6x0.Migrate(ctx, app.GovKeeper); err != nil {
+				return fromVM, err
+			}
```

Comment:
```suggestion
			}
			if err := app.LeverageKeeper.SetParams(leveragetypes.DefaultParams()); err != nil {
				return fromVM, err
			}
```
We'll need this after my params PR (but I don't want to create a merge conflict in this function by changing too much there)